### DISCUSSION TITLE
Trajectory

### DISCUSSION THREAD

**Discussion by galleon** [junzis/openap#26] [Q&A] [answered]
Hi @junzis!

Thanks for OpenAP, it seems to be a great tool. Is there a function that put a given aircraft is a certain state and then calculate 
a trajectory to reach a second state (assume both are in the cruise level). 
Or may be I have to build it myself based on ... traj ?

Cheers
Guillaume

**Comment by junzis** [comment, marked as answer]
HI @galleon, this is not included in the library at the moment. However, your case should be relatively simple to implement, with the differential equations from my paper. (my apology for the late reply).
